client/src/helpers/APICalls/uploadProfilePhoto.ts

```diff
@@ -0,0 +1,28 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+interface ImageData {
+  success: { imageURI: string };
+  error: { message: string };
+}
+
+const uploadProfilePhoto = async (images: Array<File>): Promise<ImageData> => {
+  const form_data = new FormData();
```

Comment:
`form_data` is snake case, JavaScript convention is camelCase. Please address this where applicable in the PR.